imodels/rule_set/rule_fit.py

```diff
@@ -167,15 +167,14 @@ class RuleFitRegressor(BaseEstimator, TransformerMixin):
                     number of rules will usually be lower than this due to duplicates.
     memory_par:     scale multiplier (shrinkage factor) applied to each new tree when 
                     sequentially induced. FP 2004 (Sec. 2)
-    rfmode:         'regress' for regression or 'classify' for binary classification.
     lin_standardise: If True, the linear terms will be standardised as per Friedman Sec 3.2
                     by multiplying the winsorised variable by 0.4/stdev.
     lin_trim_quantile: If lin_standardise is True, this quantile will be used to trim linear 
                     terms before standardisation.
     exp_rand_tree_size: If True, each boosted tree will have a different maximum number of 
                     terminal nodes based on an exponential distribution about tree_size. 
                     (Friedman Sec 3.3)
-    model_type:     'r': rules only; 'l': linear terms only; 'rl': both rules and linear terms
+    include_linear: Include linear terms as opposed to only rules
```

Comment:
This was a nice cleanup. Now that we have skope rules, I wonder if anyone would use this with `include_linear=False`. The code might be cleaner if we don't support this and then move decision_function into skope_rules directly. But let's leave it for now...